astropy/stats/tests/test_funcs.py

```diff
@@ -346,6 +346,19 @@ def test_mad_std():
         data = normal(5, 2, size=(100, 100))
         assert_allclose(funcs.mad_std(data), 2.0, rtol=0.05)
 
+def test_mad_std_with_axis():
+    with NumpyRNGContext(12345):
+        data = np.array([[1, 2, 3, 4],
+                         [4, 3, 2, 1]])
+        
+        #results follow data symmetry 
+        result_axis0 = np.array([ 2.22390333,  0.74130111,  0.74130111,  2.22390333])
+        result_axis1 = np.array([ 1.48260222,  1.48260222])
+
+        assert_allclose(funcs.mad_std(data, axis=0), result_axis0, rtol=0.05)  
```

Comment:
I've seldom used `assert_allclose` but do the tests break if you don't define `rtol`? You are giving 8 significant digits and the standard `rtol` is only comparing 1e-7 (7 significant digits) so theoretically the `rtol`-argument should be unnecessary. But better check if the tests pass on your computer before making another Commit.